docs/source/devguide/dev_assisted_package_creation.rst

```diff
@@ -0,0 +1,104 @@
+ .. dropdown:: Distribution Statement
+
+    | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+    | # # #
+    | # # # Author:
+    | # # # Naval Research Laboratory, Marine Meteorology Division
+    | # # #
+    | # # # This program is free software: you can redistribute it and/or modify it under
+    | # # # the terms of the NRLMMD License included with this program. This program is
+    | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+    | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+    | # # # for more details. If you did not receive the license, for more information see:
+    | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Developer Assisted Package Creation
+===================================
+
+Why We Assist With Package Creation
+-----------------------------------
+
+The GeoIPS Development team is committed to integrating a variety of geospatial
```

Comment:
This doesn't clarify _why_ we assist, just that we are "tasked" with it (by whom?)

Reply:
I don't know exactly who gave 'the order', but I know it's part of my job description as a CIRA developer for GeoIPS. Both Jeremy and John Haynes have let me know that I'll be transitioning isolated processing routines into GeoIPS, and I'm pretty sure that's part of your job description too. If I had to guess, Josh Cossuth; but I think that information is irrelevant for documentation's sake.

Reply:
I think we're tasked with helping people at CIRA but do it out of kindness for others. If we were to get to a point where we were investing a significant amount of time helping someone from outside of CIRA integrate their algorithm we would need to find a way to be funded for the work.

In other words, we're happy to help within reason, but it is still the responsibility of the algorithm developer to actually integrate their algorithm unless we're told otherwise by a funding agent.